content/en/docs/topics/kubernetes_apis.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Kubernetes APIs"
+description: "Explains Kubernetes APIs in Helm templates"
+aliases: ["docs/k8s_apis/"]
+---
+
+Helm templates specify a [Kubernetes API group](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-groups) when defining a Kubernetes object, similar to a Kubernetes manifest file. It is specified in the `apiVersion` field of the template and it identifies the API version of the object.
+
+Like all API-driven system, Kubernetes has a [deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/) for its APIs. This means that after a certain number of releases, an API version cited for deprecation will no longer work in a Kubernetes version.
+
+Helm users and chart maintainers need to be aware of Kubernetes APIs cited for deprecation to avoid issues. 
```

Comment:
`...be aware of Kubernetes APIs cited for deprecation...`
maybe should be
`...be aware of Kubernetes APIs cited as deprecated...`
or simpler
`...be aware of deprecated Kubernetes APIs...`

Reply:
Updated